src/commands/settings/AboutCommand.ts

```diff
@@ -64,7 +62,7 @@ export default class AboutCommand extends BaseCommand {
                     inline: true
                 }, {
                     name: 'Support',
-                    value: this.metadata.author.email,
+                    value: `rakinar2@onesoftnet.eu.org`,
```

Comment:
This I think we can revert it back to `this.metadata.author.email`. Because it will be easier to change stuff later.